Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -28,39 +29,55 @@ struct DeclarationsSectionTranslator: RenderSectionTranslator {
             for pair in declaration {
                 let (platforms, declaration) = pair
                 
-                let renderedTokens = declaration.declarationFragments.map { token -> DeclarationRenderSection.Token in
-                    
-                    // Create a reference if one found
-                    var reference: ResolvedTopicReference?
-                    if let preciseIdentifier = token.preciseIdentifier,
-                       let resolved = renderNodeTranslator.context.symbolIndex[preciseIdentifier] {
-                        reference = resolved
-                        
-                        // Add relationship to render references
-                        renderNodeTranslator.collectedTopicReferences.append(resolved)
-                    }
-                    
-                    // Add the declaration token
-                    return DeclarationRenderSection.Token(fragment: token, identifier: reference?.absoluteString)
-                }
-                
                 let platformNames = platforms.sorted { (lhs, rhs) -> Bool in
                     guard let lhsValue = lhs, let rhsValue = rhs else {
                         return lhs == nil
                     }
                     return lhsValue.rawValue < rhsValue.rawValue
                 }
                 
+                let renderedTokens = renderDeclarationTokens(fragments: declaration.declarationFragments, renderNodeTranslator: &renderNodeTranslator)
+                
+                // If this symbol has overloads, render their declarations as well.
+                // TODO: Handle objc overloads?
```

Comment:
Objective-C doesn't have name overloads because two methods with the same name also have the same selector. See [the archived The Objective-C Programming Language][1] if you're curious.

[1]: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40011210-CH1-SW1

That said, it'd still be good to process all the variants here to support other source languages in the future.